force_wfmanager/central_pane/plot.py

```diff
@@ -77,28 +125,50 @@ class Plot(HasStrictTraits):
     # View
     # ----
 
-    view = View(VGroup(
-        HGroup(
-            Item('x'),
-            Item('y'),
-        ),
-        UItem('_plot', editor=ComponentEditor()),
+    view = View(
         VGroup(
-            UItem('reset_plot', enabled_when='reset_enabled')
+            HGroup(
+                Item('x'),
+                Item('y'),
+                UItem('color_options'),
+            ),
+            UItem('_plot', editor=ComponentEditor()),
+            VGroup(
+                UItem('reset_plot', enabled_when='reset_enabled')
+            )
         )
-    ))
+    )
 
     def __init__(self, analysis_model, *args, **kwargs):
         self.analysis_model = analysis_model
         super(Plot, self).__init__(*args, **kwargs)
 
         self.update_data_arrays()
 
-    # Defaults
+    # Disable discrete color maps for now
+    @on_trait_change('colormap_type')
+    def update_colormap_list(self):
```

Comment:
We could remove the commented code and make an issue for this with the commented stuff there. There's also the thing of currently having a one item dropdown menu now we took out the discrete maps. 
I'm never sure whether it's better to leave in things that are going to be added to in future or take them out and have each PR be self-contained. I'd be happy to leave the dropdown as it is, maybe it'll be good as a reminder to add the discrete maps at some point?

Reply:
I agree (cf. conversation below), it might be too much of a pain to support this and would certainly complicate the analysis model.